.changes/next-release/bugfix-AmazonS3-36e9d26.json

```diff
@@ -0,0 +1,5 @@
+{
+    "type": "bugfix",
+    "category": "Amazon S3",
```

Comment:
minor: we probably should add another entry with category `AWS Java SDK for v2` since we've made changes in core as well.

Reply:
The core change is just a new class to support the S3 change (and possibly other similar changes in the future in other clients if it ever becomes necessary). Do you think it's still necessary to add a release note for a new utility class in this content?

Reply:
Oh okay, I thought you made other changes as well. Then no need to create one for core.